benchexec/test_util.py

```diff
@@ -5,140 +5,129 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-import sys
-import unittest
+import pytest
 from benchexec.util import ProcessExitCode
 import tempfile
 import os
 import stat
+import shutil
```

Comment:
I had to use shell utilities as I was getting errors multiple time i.e. `FAILED test_util.py::TestRmtree::test_writable_file - AssertionError: Failed to remove directory with file
FAILED test_util.py::TestRmtree::test_writable_dir - AssertionError: Failed to remove directory with child directory
FAILED test_util.py::TestRmtree::test_nonwritable_file - AssertionError: Failed to remove directory with non-writable file`